code/src/Album.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+/*import data from './data.json'*/
+import "./Components/album.css";
+
+/*console.log(data)
+console.log(data.albums.items[1])*/
+
+export const Album = (props) => {
+    return (
+        <div className="album-cover">
+            {/* only album cover and icons*/}
+            <div className="cover-and-icons">    
+                <div className="icons">
+                    <img className="heart-icon" src="icons/heart.svg" alt="Heart"/>
+                    <img className="play-icon" src="icons/play.svg" alt="Play"/>
+                    <img className="dots-icon" src="icons/dots.svg" alt="Dots"/>
+                </div>
```

Comment:
This feels like a good position for creating a component. Maybe called `Icon` that takes props called `className`, `src` and `alt`?